taxi/forms.py

```diff
@@ -0,0 +1,31 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.core.validators import MinLengthValidator, MaxLengthValidator
+
+from taxi.models import Driver
+
+
+class DriverLicenseUpdate(forms.ModelForm):
```

Comment:
It's not correct to use the default form for creating Driver(User). 

with DriverForm:
![image](https://user-images.githubusercontent.com/72562232/164517984-a8364c48-5141-46ae-9ab6-f756357dc812.png)
without DriverForm:
![image](https://user-images.githubusercontent.com/72562232/164518165-edebc438-408b-4629-a365-bd74b5fce5ce.png)

Many redundant fields!
Best practice:
```
class DriverForm(UserCreationForm):
        ...
```

Reply:
Thanks for your feedback!